### DISCUSSION TITLE
no such element: Unable to locate element

### DISCUSSION THREAD

**Discussion by OneAmethyst** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#420] [Q&A]
![image](https://user-images.githubusercontent.com/20665985/222860872-16cba71d-8403-4499-906c-f3cc34ef48ff.png)

This error is reported when I close VPN
![image](https://user-images.githubusercontent.com/20665985/222862105-5ad58a24-e062-4b57-a66b-52b41fa31822.png)

how to solve thie problem?

**Comment by cullzie** [comment]
Hi @OneAmethyst, 
Please use the CLI enrollment as it is much more stable and more likely to be maintained.
`udemy_enroller`
At the moment the browser remains only for legacy but it is likely in a state of disrepair.

There is a new version of the library released to pypi as well so please make sure to update to that for the best experience.
